src/layman/user/rest_user.py

```diff
@@ -0,0 +1,56 @@
+from flask import current_app as app, g, Blueprint, jsonify, request
+from layman.authn import authenticate
+from layman import util as layman_util, LaymanError
+from layman.common.prime_db_schema.users import get_user_infos
+from layman.layer import LAYER_TYPE, util as layer_util
+from layman.map import MAP_TYPE, util as map_util
+from .util import get_user_profile, delete_user
+
+
+bp = Blueprint('rest_user', __name__)
+
+
+@bp.before_request
+@authenticate
+def before_request():
+    pass
+
+
+@bp.route('/<username>', methods=['DELETE'])
+def delete(username):
+    app.logger.info(f"DELETE User request for username={username} by user={g.user}")
+    if username not in get_user_infos(username):
+        app.logger.warning(f"User {username} does not exist.")
+        raise LaymanError(57)
+    actor_profile = get_user_profile(g.user)
+    actor_profile_name = actor_profile.get('username')
```

Comment:
I've realized it's better to use method `get_authn_username` instead of `get_user_profile`. It's faster and it returns also values for ANONYM_USER and NONAME_USER.
So lines 25-26 can be rewritten to
```
actor_name = get_authn_username()
```

Also `actor_name` makes more sense than `actor_profile_name` (I did not realize it yesterday)